### PR TITLE
AWS deployment documentation

### PR DESCRIPTION
**Pull request by inercia**
Documentation for deployments in AWS.

Documents the features provided by SUSE/skuba#396
Related to SUSE/avant-garde#267
See also SUSE/avant-garde#242

### CODE REVIEW

**Comment by inercia** [comment]
@r0ckarong I'm not sure if I should document the product registration part (ie, the `SUSEConnect` stuff). It is done automatically from the Terraform scripts, but it seems there is a specific section in the docs for doing this manually...

**Comment by r0ckarong** [comment]
> @r0ckarong I'm not sure if I should document the product registration part (ie, the `SUSEConnect` stuff). It is done automatically from the Terraform scripts, but it seems there is a specific section in the docs for doing this manually...

Ultimately we need to describe all manual steps too. So please just add it if it's not too much effort.

File: adoc/deployment-aws.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
+== Deployment on Amazon AWS
+
+.Preparation Required
+[NOTE]
+You must have completed <<deployment.preparations>> to proceed.
+
+You will use {tf} to deploy the required master and worker cluster nodes (plus a load
+balancer) to {soc} and then use the `skuba` tool to bootstrap the {kube} cluster on
```
Comment (r0ckarong) [inline adoc/deployment-aws.adoc]: Why does this state {soc}? Do we run SOC on top of AWS?

File: adoc/deployment-aws.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+. Download the AWS credentials
+.. Log in to the AWS console.
+.. Click on your username in the upper right hand corner to reveal the dropdown menu.
+.. Click on My Security Credentials.
+.. Click Create Access Key on the Security Credentials tab.
+.. Note down the newly created _Access_ and _Secret_ keys.
+
+=== Deploying the cluster nodes
+
+. Find the {tf} template files for {soc} in `/usr/share/caasp/terraform/aws`.
```
Comment (r0ckarong) [inline adoc/deployment-aws.adoc]: Again `{soc}` the referred filename is `/aws`

File: adoc/deployment-aws.adoc [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+====
+The IP addresses of the generated machines will be displayed in the terraform
+output during the cluster node deployment. You need these IP addresses to
+deploy {productname} to the cluster.
+
+If you need to find an IP addresses later on you can run `terraform output` within
+the directory you performed the deployment from `~/my-deployment` directory or
+perform the following steps:
+
+. Log in to {soc} and click on menu:Load Balancing[Load Balancers]. Find the one with the
```
Comment (r0ckarong) [inline adoc/deployment-aws.adoc]: Again reference to `{soc}`. How do you deploy a load balancer for AWS?

File: adoc/deployment-aws.adoc (hunk 1 above)
Comment (inercia) [reply adoc/deployment-aws.adoc]: Fixed.

File: adoc/deployment-aws.adoc (hunk 2 above)
Comment (inercia) [reply adoc/deployment-aws.adoc]: Fixed.

File: adoc/deployment-aws.adoc (hunk 3 above)
Comment (inercia) [reply adoc/deployment-aws.adoc]: Fixed. The Load Balancer will be created ajutomatically from the Terraform code.

**Pull request merged**